lib/cdo/shared_constants.rb

```diff
@@ -2,7 +2,10 @@
 
 # This is the source of truth for a set of constants that are shared between JS
 # and ruby code. generateSharedConstants.rb is the file that processes this and
-# outputs JS. It is run via `rake build:shared_constants
+# outputs JS. It is run via `rake build:shared_constants`.
```

Comment:
Nor can I. The comment was added in #13270, so tagging its author @Bjvanminnen.

That said, the new comment still seems relevant.